lambdas/experimentcriconfig/src/main/java/uk/gov/di/ipv/core/experimentcriconfig/ExperimentCRIConfigHandler.java

```diff
@@ -0,0 +1,139 @@
+package uk.gov.di.ipv.core.experimentcriconfig;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import software.amazon.awssdk.http.urlconnection.UrlConnectionHttpClient;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.ssm.SsmClient;
+import software.amazon.lambda.powertools.parameters.ParamManager;
+import software.amazon.lambda.powertools.parameters.SSMProvider;
+import software.amazon.lambda.powertools.tracing.Tracing;
+import uk.gov.di.ipv.core.experimentcriconfig.annotations.ExcludeFromGeneratedCoverageReport;
+import uk.gov.di.ipv.core.experimentcriconfig.domain.CredentialIssuerConfig;
+import uk.gov.di.ipv.core.experimentcriconfig.domain.ErrorResponse;
+import uk.gov.di.ipv.core.experimentcriconfig.exceptions.ParseCredentialIssuerConfigException;
+import uk.gov.di.ipv.core.experimentcriconfig.helpers.ApiGatewayResponseGenerator;
+
+import java.net.URI;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@ExcludeFromGeneratedCoverageReport
+public class ExperimentCRIConfigHandler
+        implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
+
+    public static final int LOCALHOST_PORT = 4567;
+    private static final String LOCALHOST_URI = "http://localhost:" + LOCALHOST_PORT;
+    private static final String IS_LOCAL = "IS_LOCAL";
+    public static final String CREDENTIAL_ISSUERS_CONFIG_PARAM_PREFIX =
+            "CREDENTIAL_ISSUERS_CONFIG_PARAM_PREFIX";
+
+    private final SSMProvider ssmProvider;
+    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
In the interests of going quick, is it worth trying to use Gson rather than Jackson here? Something something reflection.

Reply:
Good idea! Will implement :)

Reply:
Hmm. It might mean pulling in a new dependency though. I wonder what the tradeoff is/which way is faster.

Reply:
Could be worth spinning up 2 versions of the lambda? One with jackson and one with Gson to see if there is a big difference or not?